spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/export/appoptics/AppOpticsMetricsExportAutoConfigurationTests.java

```diff
@@ -52,7 +52,14 @@ void autoConfiguresItsConfigAndMeterRegistry() {
 	}
 
 	@Test
-	void autoConfigurationCanBeDisabled() {
+	void autoConfigurationCanBeDisabledWithGlobalEnabledProperty() {
```

Comment:
I will do this same mod to the other export config tests once I know we are good w/ the direction of this proposal.